src/compiler/core.ts

```diff
@@ -2,8 +2,9 @@
 /// <reference path="performance.ts" />
 
 namespace ts {
+    export const versionMajorMinor = "2.5";
     /** The version of the TypeScript compiler release */
-    export const version = "2.5.0";
+    export const version = `${versionMajorMinor}.0`;

```

Comment:
Ah, see that now, thanks a lot!

I wonder whether moving complexity out of testing code into the compiler is worthy. Especially, now it's surfaced in the API in a confusing way (can we still assume normal version is dot-joined numbers?). And of course readability and text-searchability suffers.